tests/Exception/Http/HttpTest.php

```diff
@@ -96,10 +97,100 @@ public static function dataGetClass() {
 				'code'     => '404',
 				'expected' => '\\' . Status404::class,
 			],
-			'integer 422: class does not exist' => [
-				'code'     => 422,
+			'integer 460: class does not exist' => [
+				'code'     => 460,
 				'expected' => $default,
 			],
 		];
 	}
+
+
+	/**
+	 * Test whether all HTTP exception class can be instantiated.
+	 *
+	 * @dataProvider dataInstantiateHttpException
+	 *
+	 * @param int $code HTTP status code.
```

Comment:
```suggestion
	 * @param int $status_code HTTP status code.
```
Also missing `@param` tag for `$expected_exception_class`